release/xgboost_tests/app_config_gpu.yaml

```diff
@@ -15,6 +15,7 @@ post_build_cmds:
   - sudo rm -rf /home/ray/anaconda3/lib/python3.7/site-packages/numpy || true
   - pip3 install numpy || true
   - pip3 install -U {{ env["RAY_WHEELS"] | default("ray") }}
+  - pip3 install -U "xgboost_ray" petastorm  # Install latest releases
```

Comment:
I think we want to be testing using xgboost_ray master right?

Reply:
Do we? I thought users would usually just use the latest release in most cases (and release circles are very quick for these libraries).

Should we maybe just test both?